packages/node_modules/@cerebral/router/index.d.ts

```diff
@@ -0,0 +1,31 @@
+export interface StringRoutes {
```

Comment:
We decided not to make these exports a public API so I thought they should not go into the `router.ts` file.

Here is my current router definition:

```js
const router =
Router
( { routes:
    [ { path: '/'
      , map:
        { score: compute
          ( state`score`, prepareUrlParams )
        }
      , rmap:
        { 'score': compute 
          ( props`score`
          , ( opts: Partial<ScoreUrlType>
            ) => {
              if ( ! opts ) return baseScoreState
              return validateUrlParams ( baseScoreState, opts )
            }
          )
        }
      }
    ]
    , query: true
    , onlyHash: true
  }
)
```